CONTRIBUTORS.md

```diff
@@ -0,0 +1,12 @@
+# Contributors
+
+**By adding your name, email address, and copyright date below, you understand and agree to the terms of the [Developer's Certificate of Origin](https://developercertificate.org/) (DCO) version 1.1, and you are submitting all contributions you make to this Project pursuant to the terms described in [LICENSE.md](LICENSE.md).**
+
+## Signed-off-by
+
+- Copyright 2023-2024 Mathew Ferreira, mferreira@rite-solutions.com
```

Comment:
I think we need to represent the whole history of the project here, so please add Carl 2021-2024, and Chris 2022-2024